9.4/Dockerfile

```diff
@@ -16,14 +16,25 @@ ENV POSTGRESQL_VERSION=9.4 \
     HOME=/var/lib/pgsql \
     PGUSER=postgres
 
-LABEL io.k8s.description="PostgreSQL is an advanced Object-Relational database management system" \
+ENV SUMMARY="Advanced Object-Relational database management system" \
```

Comment:
We should keep PostgreSQL keyword in the summary, for cases it is the only field available.. So, I'd rather use something like `PostgreSQL object-relational database server`